src/main/java/edu/cornell/kfs/paymentworks/service/PaymentWorksUtilityService.java

```diff
@@ -35,4 +37,18 @@
 	 * @return
 	 */
 	String pojoToJsonString(Object object);
+	
+	/**
+	 * Determines if the vendor should be uploaded to PaymentWorks
+	 * @param vendorDetail
+	 * @return
+	 */
+	boolean shouldVendorBeSentToPaymentWorks(VendorDetail vendorDetail);
+	
+	/**
+	 * Determines if the vendor should be uploaded to PaymentWorks
```

Comment:
I think this comment should be removed. the method name is self explanatory. the comment does not add any value here.